### ISSUE TITLE
Problem of transition effect on file open

### ISSUE THREAD

**Issue by Peter2121** [Obsidian-StudiosInc/ecrire#50]
##### Description
When the "Animate open" option is active - it is impossible to open file from command line. Probably, at the moment of calling `elm_transit_go` function the `ecrire` window is not completely initialized.

##### Steps to reproduce
1. Start `ecrire`
2. Activate option "Animate open"
3. Close `ecrire`
4. Open file from command line with `ecrire`:
`ecrire test.txt`

When `ecrire` is already started, the file can be open correctly.

##### System information
- EFL Version: GIT
- E Version: GIT
- X or Wayland: X

The issue can be reproduced with last releases of EFL and E too.

**Comment by wltjr** [comment]
Hi @Peter2121, that is definitely a problem. I can confirm the unwanted behavior you reported. It is kind of funny, the document opens, you see it, but then it goes away and you have a blank document. Like, where did my document go? I will look into a resolution ASAP. Thanks again for the report!

**Comment by wltjr** [comment]
@Peter2121, I made some changes and added a animation delay when opening a document during ecrire start. That resolves the issue reported, but leaving open for additional confirmation. I may need to increase the delay. I noticed, if the delay was less than a quarter of a second, the problem remained. At a quarter of a second or greater, the issue does not occur. Hopefully, that does not vary from machine to machine.
